x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.test.ts

```diff
@@ -67,6 +70,13 @@ const defaultCallClusterMock = jest.fn().mockResolvedValue({
   },
 });
 
+const getMockFetchClients = (mockedCallCluster: jest.Mock) => {
+  return {
+    callCluster: mockedCallCluster,
+    esClient: elasticsearchServiceMock.createClusterClient().asInternalUser,
+  };
```

Comment:
Let's extend `createCollectorFetchClientsMock()` so we don't need to revisit these tests when we extend the clients/context/whatever-name-we-come-up-with 😅).

```suggestion
  return {
    ...createCollectorFetchClientsMock(),
    callCluster: mockedCallCluster,
  };
```